### PR TITLE
Remove Device Info from public API + Fix UnitTest CI

### PR DESCRIPTION
**Pull request by PravinPK**
Remove passing the device information through the Event, rather device information can be obtain at the time of registration call

### CODE REVIEW

File: AEPCampaignClassic/Sources/CampaignClassic+PublicAPI.swift [hunk 1]
```diff
@@ -25,14 +25,10 @@ public extension CampaignClassic {
...
     static func registerDevice(token: Data, userKey: String, additionalParameters: [String: Any]?) {
-        let deviceInfo = [CampaignClassicConstants.EventDataKeys.CampaignClassic.DEVICE_INFO_KEY_DEVICE_NAME: UIDevice.current.name,
-                          CampaignClassicConstants.EventDataKeys.CampaignClassic.DEVICE_INFO_KEY_DEVICE_MODEL: UIDevice.current.model,
-                          CampaignClassicConstants.EventDataKeys.CampaignClassic.DEVICE_INFO_KEY_OS_NAME: UIDevice.current.systemName]
 
         var eventData = [CampaignClassicConstants.EventDataKeys.CampaignClassic.REGISTER_DEVICE: true,
                          CampaignClassicConstants.EventDataKeys.CampaignClassic.DEVICE_TOKEN: token.hexDescription,
-                         CampaignClassicConstants.EventDataKeys.CampaignClassic.USER_KEY: userKey,
-                         CampaignClassicConstants.EventDataKeys.CampaignClassic.DEVICE_INFO: deviceInfo] as [String: Any]
+                         CampaignClassicConstants.EventDataKeys.CampaignClassic.USER_KEY: userKey] as [String: Any]
```
Comment (swarna04) [inline AEPCampaignClassic/Sources/CampaignClassic+PublicAPI.swift:31]: Why is `userKey` being cast to [String: Any]?

**Review by swarna04 — APPROVED** [review]
@PravinPK Please feel free to merge after addressing my comment here.

File: AEPCampaignClassic/Sources/CampaignClassic+PublicAPI.swift (hunk 1 above)
Comment (PravinPK) [reply AEPCampaignClassic/Sources/CampaignClassic+PublicAPI.swift:31]: It's not the `userKey`, its the whole eventData .
The code is something like
```
let eventData = [... : ...] as [String : Any]
```

Comment (swarna04) [reply AEPCampaignClassic/Sources/CampaignClassic+PublicAPI.swift:31]: ah my bad! :P

**Pull request merged**